apps/gro/fields/address.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 module.exports = {
-  address: {
-    className: ['address']
+  'address-textarea': {
+    validate: ['required'],
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens']
```

Comment:
https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L21 does default here, then additional underneath. This isn't part of this PR though, just renamed the file.